### PR TITLE
prettify the demo

### PR DESCRIPTION
**Pull request by notwaldorf**
Here's a fancier demo. I am worried we are covering too many examples? What do you think?
/cc @addyosmani for input

(I've also updated the tests to the wct of the future, etc, etc)

screen shot 2015-12-03 at 3 24 31 pm
screen shot 2015-12-03 at 3 24 41 pm
screen shot 2015-12-03 at 3 24 50 pm
screen shot 2015-12-03 at 3 24 56 pm

### CODE REVIEW

File: test/paper-input.html [hunk 1]
```diff
@@ -271,7 +268,7 @@
           });
         });
       });
-     
+
```
Comment (cdata) [inline test/paper-input.html]: Is this line even needed?

File: test/paper-textarea.html [hunk 2]
```diff
@@ -14,19 +14,15 @@
...
 
   <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
 
   <script src="../../web-component-tester/browser.js"></script>
-  <script src="../../test-fixture/test-fixture-mocha.js"></script>
-
   <script src="../../iron-test-helpers/test-helpers.js"></script>
   <script src="../../iron-test-helpers/mock-interactions.js"></script>
 
-  <link rel="import" href="../../test-fixture/test-fixture.html">
```
Comment (cdata) [inline test/paper-textarea.html]: :beers:

File: demo/index.html [hunk 3]
```diff
@@ -49,82 +70,84 @@
...
+      </template>
+    </demo-snippet>
+
+    <h3>Inputs can have prefixes and suffixes</h3>
+    <demo-snippet class="horizontal">
+      <template>
+        <paper-input class="short" label="total" type="number">
+          <div prefix>$</div>
+        </paper-input>
+        <paper-input class="short" label="username">
```
Comment (abdonrd) [inline demo/index.html]: Remember add the `id="inputWithButton"` for the `clearInput()`. :)

**Comment by cdata** [comment]
I like the different examples, personally.

**Comment by cdata** [comment]
LGTM :+1: 

> one test fails in IE10
> unsure if anyone cares

**Comment by notwaldorf** [comment]
Fixed the test! (a fixture had the wrong closing tag)

**Pull request merged**